controller/search.go

```diff
@@ -286,6 +293,28 @@ func (c *SearchController) Spaces(ctx *app.SpacesSearchContext) error {
 		q = "" // Allow empty query if * specified
 	}
 
+	var sort string
+	if ctx.Sort == nil {
+		// this is the default case
+		// which returns recently updated spaces first
+		sort = SortByUpdatedAt + " DESC"
+	} else {
+		switch *ctx.Sort {
+		case "name":
+			sort = SortByName + " ASC"
+		case "-name":
+			sort = SortByName + " DESC"
+		case "owner":
+			sort = SortByOwnerID + " ASC"
+		case "-owner":
+			sort = SortByOwnerID + " DESC"
+		case "created":
+			sort = SortByCreatedAt + " ASC"
+		case "-created":
```

Comment:
Why not have a case for `"updated"`?

Reply:
Because that is default, so don't wanna change the default behavior, not sure how to proceed on that?

Reply:
I don't see why we can't have "updated_at" and the default being the same thing really.